logstash-core/lib/logstash/output_delegator.rb

```diff
@@ -27,7 +32,35 @@ def initialize(logger, klass, *args)
       w
     end
 
+
     @events_received = Concurrent::AtomicFixnum.new(0)
+
+    if threadsafe
+      @threadsafe_worker = @workers.first
+      self.define_singleton_method(:multi_receive, method(:threadsafe_multi_receive))
+    else
+      self.define_singleton_method(:multi_receive, method(:worker_multi_receive))
+    end
+  end
+
+  def warn_on_worker_override!
+    # The user has configured extra workers, but this plugin doesn't support it :(
+    if @config["workers"] && @config["workers"] > 1 && @klass.workers_not_supported?
+      message = @workers_not_supported_message
+      if message
```

Comment:
Do we really need to create a new variable `message` here we could use `@workers_not_supported_message`